openlibrary/catalog/marc/tests/test_data/xml_expect/zweibchersatir01horauoft.json

```
@@ -6,32 +6,37 @@
   "table_of_contents": [
     {
       "type": "/type/toc_item",
-      "title": "Vol. 1. Text, \u00dcbersetzung und kritischer Apparat.-  Vol. 2, pt. 1, Commentar zum ersten Buche der Satiren.-  Vol. 2, pt. 2, Commentar zum zweiten Buche der Satiren, verfasst von W.S. Teuffel."
+      "title": "Vol. 1. Text, Übersetzung und kritischer Apparat.-  Vol. 2, pt. 1, Commentar zum ersten Buche der Satiren.-  Vol. 2, pt. 2, Commentar zum zweiten Buche der Satiren, verfasst von W.S. Teuffel."
     }
   ],
-  "title": "Zwei B\u00fccher Satiren",
+  "title": "Zwei Bücher Satiren",
   "work_titles": [
     "Satirae"
   ],
-  "languages": [
-    "ger",
-    "lat"
-  ],
+  "languages": ["ger", "lat"],
   "publish_date": "1854",
   "publish_country": "ge",
   "authors": [
     {
-      "personal_name": "Horace",
       "name": "Horace",
       "entity_type": "person"
+    },
+    {
+      "name": "Kirchner, Carl Christian Jacob",
+      "birth_date": "1787",
+      "death_date": "1855",
+      "entity_type": "person",
+      "role": "tr. [and] ed"
```

```suggestion
      "role": "tr. [and] ed."
```
$e is a repeatable subfield, so the "[and]" was unnecessary here, but given the way that it was cataloged, the best we can do is preserve the period (full stop) for the abbreviation of "editor" here.